client/src/App.tsx

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Please just use this import in the index.tsx file instead of the link tag you've used below